platform-bom/pom.xml

```diff
@@ -666,6 +667,26 @@
 				<artifactId>shared-ldap</artifactId>
 				<version>${apache-shared-ldap.version}</version>
 			</dependency>
+			<dependency>
```

Comment:
This should list all Apache Geode modules that are released together, not just those that Spring Data Geode directly depends upon. Listing all of a project's modules reduces the chance of a user ending up with mixed versions.

Reply:
Yes, agreed; will fix.

Reply:
A few notes, FYI...

1. I have no idea what the `apache-geode` artifact is.  However, since I am basing the _Spring IO Platform_ on SD Kay, that includes SD Geode 2.0.2.RELEASE, which is itself based on Apache Geode 1.2.1, I have excluded this dependency since [there is no](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.apache.geode%22%20AND%20a%3A%22apache-geode%22) `org.apache.geode:apache-geode:1.2.1` dependency.

2. Also, `geode-protobuf` is new as of 1.3.0 (i.e. not in 1.2.1 yet).

3. This includes `geode-concurrency-test` as well; only in 1.3.0, not in 1.2.1.

4. Finally, I would also add that when I submitted (updated) this PR, I essentially modeled the dependencies based on what [you put](https://github.com/spring-io/platform/commit/68c8f5e7cbb10f8b032c53507adc7d6f64203c5f#diff-8c74b2e34698dac4b7de48b901352e95R536) into the Spring IO Platform [BOM file for Pivotal GemFire](https://github.com/spring-io/platform/blob/master/platform-bom/pom.xml#L500-L518); I certainly did not do this, but Pivotal GemFire dependency management is effectively the same as Apache Geode's, FYI.

Reply:
NOTE: I excluded the `org.apache.geode:geode-modules*` since, apparently, after speaking with the Pivotal GemFire team, these were never meant to be published to Maven Central.  

Additionally, I had trouble with the `geode-modules-session` dependency since they inappropriately specified a dependency to `geode-modules-session-internal`, as...

````
<dependency>
    <groupId>org.apache.geode</groupId>
    <artifactId>extensions/geode-modules-session-internal</artifactId>
    <version>1.2.1</version>
    <scope>compile</scope>
</dependency>
````

Notice the `artifactId`.  Argh!

Anyway, FYI.